recipes/mediamtx/recipe.yaml

```
@@ -0,0 +1,76 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: mediamtx
+  version: "1.14.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/bluenviron/mediamtx/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: fa388288865aec093720953af8374b759ca9519f8d20699cfa8b41281e99f372
+    target_directory: src
+  - path: service.yaml
+    target_directory: service-config
+
+build:
+  number: 0
+  script:
+    - cd src
+    - echo "v${{ version }}" > VERSION
+    - go generate ./...
+    - go-licenses save . --save_path ../library_licenses --ignore golang.org/x/sys/unix --ignore golang.org/x/crypto/internal/poly1305 --ignore golang.org/x/crypto/salsa20/salsa --ignore golang.org/x/crypto/argon2 --ignore golang.org/x/crypto/blake2b --ignore golang.org/x/sys/cpu --ignore golang.org/x/crypto/sha3 --ignore github.com/ugorji/go/codec --ignore github.com/benburkert/openpgp
```

```suggestion
    - go-licenses save . --save_path ../library_licenses --ignore golang.org/x/sys --ignore golang.org/x/crypto  --ignore github.com/ugorji/go/codec --ignore github.com/benburkert/openpgp
```
This can be simplified because `golang.org/x/sys` and `golang.org/x/crypto` are each a single repository with a single license file.

One annoying snag when we have to skip repositories like this is the licenses need to be manually added to the recipe and copied into `${SRC_DIR}/license-files` using the same directory structure made by `go-licenses`.   See here for an example of how this should look: https://github.com/conda-forge/wails-feedstock/tree/main/recipe.  It's not difficult, just tedious, especially when multiple licenses are invloved.